Purchases/Purchasing/OfferingsFactory.swift

```diff
@@ -35,8 +35,26 @@ class OfferingsFactory {
         return Offerings(offerings: offerings, currentOfferingID: currentOfferingID)
     }
 
+    func createOfferings(withProducts products: [String: ProductDetails],
+                         data: [String: Any]) -> Offerings? {
+        let productIdentifiersAndDetailsAsTuple = products.map { productIdentifier, productDetails in
+            (productIdentifier, productDetails)
+        }
+        let productDetailsByKey = Dictionary(uniqueKeysWithValues: productIdentifiersAndDetailsAsTuple)
+        return self.createOfferings(withProductDetails: productDetailsByKey, data: data)
+    }
+
     func createOffering(withProducts products: [String: SKProduct],
```

Comment:
will we still need this one?

Reply:
nope! I had kept it around it case it was needed, and it's not anymore!